content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -12,218 +12,278 @@ import GenerateRecoveryToken from '@site/content/_generate-recovery-token.md';
+To complete this guide, you need:
 
-## Requirements
+- Pomerium Core
+- A configured identity provider
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- PostgreSQL
+- Prometheus
 
-- Pomerium Enterprise requires Linux amd64/x86_64. It can manage Pomerium instances on other platforms, however.
-- Each Console instance should have at least:
-  - 4 vCPUs
-  - 8G RAM
-  - 100G of disk wherever logs are stored
-- Each Postgres instance should have at least:
-  - 4 vCPUs
-  - 8G RAM
-  - 20G for data files
+This guide runs Pomerium Core and PostgreSQL as Docker containers. If you haven't, pull the [Pomerium](https://hub.docker.com/r/pomerium/pomerium) and [PostgreSQL](https://hub.docker.com/_/postgres) Docker images from Docker Hub and complete the [Pomerium Core quickstart](/docs/quickstart) to set up a Pomerium instance that's compatible with this guide.
 
-## Install Pomerium Enterprise
+The Pomerium Enterprise Console uses an embedded Prometheus server to collect and display metrics. While you can [configure metrics](/docs/capabilities/metrics) to use an existing Prometheus server, this guide uses Pomerium’s Prometheus backend.
 
-Pomerium publishes standard OS packages for RPM and DEB based systems. The repositories require authentication via username and access key. These credentials will be issued to you during the onboarding process.
+:::tip **Note**
 
-<Tabs>
+This guide assumes you've already registered for Pomerium Enterprise and have credentials to access the private Cloudsmith Docker registry. If you haven't registered for Pomerium Enterprise, [sign up here](https://www.pomerium.com/enterprise-sales/) for a free trial.
 
-<TabItem value="deb" label="deb">
+:::
 
-1. To automatically configure the repository for Debian and Ubuntu distributions, run the following command replacing `[access-key]`:
+### Configure Docker Compose services
 
-   ```bash
-   curl -1sLf \
-   'https://dl.cloudsmith.io/[access-key]/pomerium/enterprise/setup.deb.sh' \
-   | sudo -E bash
-   ```
+In your `docker-compose.yaml` file, define a bridge network:
 
-   Or to manually configure, you can manually import the apt key, then create a new `.list` file in `/etc/apt/source.list.d`. Make sure to replace the distribution and version:
+```yaml title="docker compose"
+version: '3'
+networks:
+  main: {}
+```
 
-   ```bash
-   curl -1sLf 'https://dl.cloudsmith.io/[access-key]/pomerium/enterprise/gpg.B1D0324399CB9BC3.key' | apt-key add -
+All the services defined in your Docker Compose config file will communicate over the `main` network.
+
+### Configure Pomerium Core
+
+Add the `pomerium` image to your `docker-compose.yaml` file:
+
+```yaml title="pomerium service"
+version: '3'
+networks:
+  main: {}
+services:
+  pomerium:
+    image: pomerium/pomerium:latest
+    volumes:
```

Comment:
suggestion: 
```
    volumes:
        - ./config.yaml:/pomerium/config.yaml:ro # Mount your config file read-only
        - .data:/data:rw # Mount persistent persistent data storage
```

IMO this should be in the [main Quick-Start](https://www.pomerium.com/docs/quickstart) also.